### PR TITLE
Do not produce timestamped reports by default

### LINKED ISSUE #318: Do not produce timestamped reports by default (PIT 1.9.6)
Timestamped reports are no longer produced by default by PIT. The plugin should also take it as a default behavior.

https://github.com/hcoles/pitest/pull/1085

### PR DESCRIPTION
**Pull request by SidB3**
Fixes #318

### CODE REVIEW

File: src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy [hunk 1]
```diff
@@ -362,7 +362,7 @@ class PitestTask extends JavaExec {
         map['includedTestMethods'] = optionalCollectionAsString(includedTestMethods)
         map['sourceDirs'] = (getSourceDirs().files*.absolutePath)?.join(',')
         map['detectInlinedCode'] = optionalPropertyAsString(detectInlinedCode)
-        map['timestampedReports'] = optionalPropertyAsString(timestampedReports)
+        map['timestampedReports'] = optionalPropertyAsString(timestampedReports) ?: 'false'
```
Comment (szpak) [inline src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy:365]: It's not the best place for that kind of logic. PitestTask just passes the values. The default values can be set in [`PitestPlugin`](https://github.com/szpak/gradle-pitest-plugin/blob/12857274f550d5a21d570621934bb86d5af16c0f/src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy#L114).

**Review by szpak — CHANGES_REQUESTED** [review]
Thanks for your PR. I had one suggestion to your code.

File: src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy (hunk 1 above)
Comment (SidB3) [reply src/main/groovy/info/solidsoft/gradle/pitest/PitestTask.groovy]: Thanks for clarifying. Updated the PR.

**Review by szpak — APPROVED** [review]

**Comment by szpak** [comment]
@SidB3 I took a look at https://github.com/hcoles/pitest/pull/1085/ and PIT 1.9.6 switched to do not produce timestamped reports by default. As a result, even the current plugin version should not do it as well (without any changes). Have you encountered a different behavior?

Maybe you were misled by my imprecise issue #318 description (which misled me as well...).

**Comment by SidB3** [comment]
You are correct @szpak . I just tried it out and 1.9.6 defaults to non time stamped reports. Closing the PR.

**Comment by szpak** [comment]
Anyway, thanks for your effort and sorry for the misleading issue description :-/.

**Pull request closed without merging**